src/ia-account-settings.ts

```diff
@@ -0,0 +1,897 @@
+            <ia-pic-uploader
+              identifier=${this.userData.identifier}
+              picture="${this.profilePicture}"
+              ?lookingAtMyAccount=${true}
+              type="compact"
+              @fileChanged=${() => {
+                this.changeSaveButtonState();
+              }}
+              @fileUploaded=${() => {
+                this.profilePictureUploaded();
+              }}
+            ></ia-pic-uploader>
+          </div>
+
+          <div class="form-element ">
+            <label for="screenname">
+              Change screenname <small>(will not change user id)</small>
+            </label>
+            <input
+              type="text"
+              class="form-control"
+              id="screenname"
+              name="screenname"
+              .value="${this.userData.screenname}"
+              @input=${this.setScreenname}
+              @blur=${this.validateScreenname}
+            />
+            <span class="error-field">${this.fieldsError.screenname}</span>
+          </div>
+
+          <div class="form-element">
+            <label for="email">
+              Change email <small>(verification will be required)</small>
+            </label>
+            <input
+              type="email"
+              class="form-control"
+              id="email"
+              name="email"
+              .value="${this.userData.email}"
+              @input=${this.setEmail}
+              @blur=${this.validateEmail}
+            />
+            <span class="error-field">${this.fieldsError.email}</span>
+          </div>
+
+          <div class="form-element">
+            <label for="password">
+              Change Internet Archive / Open Library password
+            </label>
+            <input
+              type="password"
+              class="form-control"
+              id="password"
+              name="password"
+              autocomplete="new-password"
+              @input=${this.setPassword}
+              @blur=${this.validatePassword}
+            />
+            <input
+              type="image"
+              class="password-icon"
+              src="https://archive.org/images/eye-crossed.svg"
+              alt="View text"
+              @click=${(e: Event) => {
+                preventDefaultAndStopEvent(e);
+                togglePassword(e, this.passwordField as HTMLInputElement);
+              }}
+            />
+            <span class="error-field">${this.fieldsError.password}</span>
+          </div>
+
+          <div class="form-element">
+            <label>Set borrow history</label>
+            <input
+              type="checkbox"
+              id="borrow-history"
+              name="borrow-history ${this.loanHistoryFlag}"
+              .checked=${this.loanHistoryFlag === 'public' ||
+              this.loanHistoryFlag === true}
+              @click=${this.setBorrowHistory}
+            />
+            <label for="borrow-history"> Visible to the public</label>
```

Comment:
Suggestion: Move the `for="borrow-history"` to the first label (the one that identifies what the checkbox is for).

Reply:
The first label is heading and the second is that identify the checkbox. so we are good here.

Reply:
Ah okay, I misunderstood the meaning of the checkbox. Sounds good 👍